packages/app/src/app/pages/common/ErrorBoundary/types.ts

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Location } from 'history';
+
+export type ErrorInfo = {
+  componentStack: string;
+};
+
+export interface IFallbackComponentProps {
+  error?: Error;
+  trace?: string;
+}
+
+export interface IErrorBoundaryProps {
+  children?: React.ReactNode;
```

Comment:
@Saeris I think it would be a bit less verbose when you just delete the `React.` part here

```suggestion
  children?: ReactNode;
```

Reply:
We're already using the long syntax elsewhere in the codebase and I'd prefer to stay consistent with other TypeScript examples